components/Demo/Demo.tsx

```diff
@@ -49,24 +51,112 @@ const Demo = () => {
             <Input
-              id='address'
-              name='address'
-              type='address'
-              variant='filled'
+              id="address"
+              name="address"
+              type="address"
+              placeholder="Enter Wallet Address"
               onChange={formik.handleChange}
               value={formik.values.address}
               disabled={isLoading}
+              variant="filled"
+              borderRadius={14}
+              marginBottom={4}
+              _placeholder={{ color: '#2A6462' }}
+              textColor={'#74FBF7'}
+            />
+            <FormLabel htmlFor="chainId">Chain Id</FormLabel>
+            <Input
+              id="chainId"
+              name="chainId"
+              type="number"
+              placeholder="Enter Chain ID"
+              onChange={formik.handleChange}
+              value={formik.values.chainId}
+              disabled={isLoading}
+              variant="filled"
+              borderRadius={14}
+              marginBottom={4}
+              _placeholder={{ color: '#2A6462' }}
+              textColor={'#74FBF7'}
+            />
+            <FormLabel htmlFor="type">Token Type</FormLabel>
+            <Input
+              id="type"
+              name="type"
+              type="text"
+              placeholder="Enter Token Type"
+              onChange={formik.handleChange}
+              value={formik.values.type}
+              disabled={isLoading}
+              variant="filled"
+              borderRadius={14}
+              marginBottom={4}
+              _placeholder={{ color: '#2A6462' }}
+              textColor={'#74FBF7'}
+            />
+            <FormLabel htmlFor="contractAddress">
+              Token Contract Address
+            </FormLabel>
+            <Input
+              id="contractAddress"
+              name="contractAddress"
+              type="address"
+              placeholder="Enter Token Contract Address"
+              onChange={formik.handleChange}
+              value={formik.values.contractAddress}
+              disabled={isLoading}
+              variant="filled"
+              borderRadius={14}
+              marginBottom={4}
+              _placeholder={{ color: '#2A6462' }}
+              textColor={'#74FBF7'}
+            />
+            <FormLabel htmlFor="minToken">Minimum Tokens in Wallet</FormLabel>
+            <Input
+              id="minToken"
+              name="minToken"
+              type="number"
+              placeholder="Enter Minimum Tokens in Wallet"
+              onChange={formik.handleChange}
+              value={formik.values.minToken}
+              disabled={isLoading}
+              variant="filled"
+              borderRadius={14}
+              marginBottom={4}
+              _placeholder={{ color: '#2A6462' }}
+              textColor={'#74FBF7'}
             />
+
+            {/* <FormLabel htmlFor="maxToken">Maximum Tokens in Wallet (optional)</FormLabel>
```

Comment:
Any reason for this to be commented?

Reply:
I suggest not keeping `maxToken` just to keep things simple. There are many other properties like `maxToken` which can be configured. 
If you agree too, then I can remove this code.

Reply:
Ah okok, yeah lets leave it out for now, can easily be added later ;)